runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -204,4 +204,17 @@
   ExecutionMode getExecutionModeForBatch();
 
   void setExecutionModeForBatch(ExecutionMode executionMode);
+
+  @Description("Savepoint restore path.")
+  String getSavepointPath();
```

Comment:
Maybe extend the description a bit, also default set to `""`?

    If specified, attempts to restore the streaming pipeline from the provided Savepoint path.

Reply:
Would be nice to test the default in `PipelineOptionstest#testdefaults`

Reply:
Done!